sql/core/src/test/resources/sql-tests/inputs/udf/pgSQL/udf-aggregates_part2.sql

```diff
@@ -0,0 +1,232 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- AGGREGATES [Part 2]
+-- https://github.com/postgres/postgres/blob/REL_12_BETA1/src/test/regress/sql/aggregates.sql#L145-L350
```

Comment:
Please use `REL_12_BETA2` here. I guess `REL_12_BETA2` will be the same with `REL_12_BETA1`.
If not, please make this up-to-date.

Reply:
Seems like there's one more case. Let me make a small PR separately to address this comment for `aggregates_part2.sql` as well.

Reply:
+1.